packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -710,6 +710,52 @@ class DropdownButton<T> extends StatefulWidget {
   /// The text style to use for text in the dropdown button and the dropdown
   /// menu that appears when you tap the button.
   ///
+  /// To use a separate text style for the dropdown button, consider using
```

Comment:
dropdown button => selected item when it's displayed within the dropdown button,